docs/components/camera/transform.md

```diff
@@ -84,7 +84,7 @@ Then remove and fill in the attributes as applicable to your camera, according t
   "type": "camera",
   "namespace": "rdk",
   "attributes" : {
-    "source" : "<your-camera-name>",
+    "source" : "<your-source-camera-name>",
```

Comment:
figured this was confusing with your-camera-name also being prompted above for the transform camera itself's name